### ISSUE TITLE
Error while trying to enable the addon on Blender

### ISSUE THREAD

**Issue by yohan-bornachot** [kugelrund/mesh_segmentation#6]
Hello,

First of all, thank you for your work, I've dived into your code and it seems to be very well built!
Still, I've tried to install the add-on on Blender 2.83, and had to modify a little bit the addon to make it compatible with the named version. Unfortunately, the addon doesn't want to be enabled and I get the following error :
"OSError :  %1 is not a Win32 valid application"

Also, I tried to install an older version of Blender to see if it was working, but I had the same problem...
I think that it is potentially related to a version problem of Scipy and/or Numpy in the lib folder, or even my python version (3.8)...

Could you please tell me what version of the different libraries you use and your python version? Thank you in advance for your answer!

Kind regards,

![ErrorBlenderWhileEnablingAddon](https://user-images.githubusercontent.com/48725625/86592641-254ba180-bf94-11ea-840c-6cfca4af27ac.PNG)

**Comment by kugelrund** [comment]
Hey,

I just migrated the add-on to the new 2.8 API, so you can use it with Blender 2.83 now. I also updated the installation instructions for SciPy, see the readme on the main page. If you follow those steps there should be no possibility for any version conflicts.

Feel free to ask here if the instructions in the readme are not clear.

**Comment by yohan-bornachot** [comment]
Hello again,

Thank you for your fast answer, that is very nice of you!
I had to uninstall Blender, reinstall it and install scipy with the cmd in administrator mode as you said in the new readme file.
And it finally worked! I was able to enable the addon in Blender!
So thank you once again! I am eager to test it concretely once I will be able to import my mesh files in Blender (I'm dealing with some special format...)

Kind regards,
